src/test/kotlin/com/metalbear/mirrord/MirrordPluginTest.kt

```diff
@@ -112,11 +119,16 @@ internal class MirrordPluginTest {
             }
 
             step("Open `app.py`") {
-                openFileByName("app.py")
-
-                editorTabs {
+                with(projectViewTree) {
```

Comment:
I remember one CI run where the project tree was not visible initially. That's why I used the `Search Everywhere` option. Does this handle the case?

Reply:
can you send me the trace log for that run?

Reply:
I think it's this one [here](https://github.com/metalbear-co/mirrord-intellij/actions/runs/5578106964/job/15105710729)
There's also a video from that (`testMirrordFlow_recording_2023_17_07_16_50_52.avi`)

Reply:
That happens because probably your action is not DumbAware

Reply:
But there was no indexing going on